tools/topology/sof/pipe-waves-playback.m4

```diff
@@ -0,0 +1,121 @@
+# Low Latency Passthrough with codec_adapter Pipeline and PCM
+# codec_adapter instantiates Waves codec
+#
```

Comment:
I wonder what is the utility of `pipe-codec-adapter-playback.m4` if we create this kind of files for each new type of codec.

Reply:
Ah good point that might be the better solution, good point for the generic pipeline, demux will likely sill need copying.